site/src/components/RichParameterInput/RichParameterInput.tsx

```diff
@@ -154,6 +155,34 @@ const RichParameterField: React.FC<RichParameterInputProps> = ({
     )
   }
 
+  if (parameter.type === "list(string)") {
+    let values: string[] = []
+
+    if (parameterValue) {
+      try {
+        values = JSON.parse(parameterValue) as string[]
+      } catch (e) {
+        console.error("Error parsing list(string) parameter", e)
+      }
+    }
+
+    return (
+      <MultiTextField
+        label={props.label as string}
+        values={values}
+        onChange={(values) => {
+          try {
+            const value = JSON.stringify(values)
+            setParameterValue(value)
+            onChange(value)
+          } catch (e) {
+            console.error("Error on change of list(string) parameter", e)
```

Comment:
Do we need to do anything else with error handling here or on line 165?

Reply:
This is something we control but in case it gets an error, I added a message to help a bit more.